includes/specials/SpecialNewimages.php

```diff
@@ -1,4 +1,8 @@
 <?php
+
+// NOTE: This file is not in use. See WikiaNewFiles.class.php instead
+
+
```

Comment:
Excellent question. It's a core MW one (+ our modifications, that are 
not in use anymore). I'm not sure what's our policy in such cases. 
Should we actually revert it to the original MW one and maybe add a die 
on top it plus this comment for anyone trying to work with it later? 
@macbre @owend?

On 12/07/2015 11:35 AM, Damon Snyder wrote:
> In includes/specials/SpecialNewimages.php
> <https://github.com/Wikia/app/pull/9248#discussion_r46865396>:
>
>> @@ -1,4 +1,8 @@
>>  <?php
>> +
>> +// NOTE: This file is not in use. See WikiaNewFiles.class.php instead
>> +
>> +
>
> Should we delete the file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/9248/files#r46865396>.
>